udf-examples/pom.xml

```diff
@@ -32,13 +32,6 @@
   <version>22.02.0-SNAPSHOT</version>
```

Comment:
I don't understand why this pom is still here.  It claims to be the UDF examples artifact, but supposedly we're moving that jar to another repository?  Also I'm not a fan of having a circular dependency in this repository where this project depends on the UDF examples jar to test (but oddly, those tests are not associated with the UDF examples code that was moved?), and the external UDF examples project depends on this project to build.

IMO everything associated with udf-examples should be moved to the external repository, code _and_ tests.  If the desire is to ensure UDF example tests are passing, then we can always update premerge and nightly build scripts to build and locally install the artifacts from this repository then subsequently build and test the udf-examples project.